mmv1/templates/terraform/constants/interconnect.go.erb

```diff
@@ -0,0 +1,3 @@
+func InterconnectTypeDiffSuppress(k, old, new string, d *schema.ResourceData) bool {
+    return old == "DEDICATED" && new == "IT_PRIVATE"
```

Comment:
```suggestion
    return old == "IT_PRIVATE" && new == "DEDICATED"
```

Reply:
I had initially tried with suggested code, it didn't work. 
OLD was DEDICATED and NEW was IT_PRIVATE, after the code changed to the current code, it worked.

Reply:
The test `TestAccComputeInterconnect_computeInterconnectBasicTestExample` still failed with [permadiff](https://fff31dccb52217de7e62f72b6f6a648527ec25dd2f78d2187aab9fc-apidata.googleusercontent.com/download/storage/v1/b/ci-vcr-logs/o/beta%2Frefs%2Fheads%2Fauto-pr-11176%2Fartifacts%2Fd1a40a59-50f6-4ee8-bc1f-da0545968f3d%2Fbuild-log%2Frecording_build%2FTestAccComputeInterconnect_computeInterconnectBasicTestExample_recording_test.log?jk=AYBlUPB91QtWrT0tTlWgeCpeQfOvzKBxp5-UF822ho-OXl8H5P-1Eu-TFEMrAIj49UESp7HV59-kgWtOV0wW6Rt5X9IzNI6MvJo-2YxgG8AFG9dx5y4c2KLQrjIWe-ml8qXnd9xHD8us3j45804UzSt-sqJRsO8l3f4wTWwH7KfiyJSDgCWrKYSF7_MarS7CXV7YusudUtZPnair_3jjM8sZ1b7bqGhCjMkjG_M7HrE3w1TZRqEm-80eZDU8mzJQpPbBt40KRJfZRdDcA2QR8lG_IkHKOxA6K-ZI6gZ4oyVLuOTX5RHtlcNTAsPhvJfK8Ez7VsKjP4MBhs7iFdZuD34qq21iXYZjmZVgD-d57PXzYweIuLXTO3gJESaFcJHKHMm3xkSLkDCtSIVRrrJNC9-fzS2q7XuTsJo82FDHLW0zw5D-WXFDNjrhj5xGX-cGabPIBSGhiVuexYOtsuHLk7rs2rEsaRrlKaS2gQxFAXerFXb-vNJg21cU7yvLYfmFrNrarynvy-pAwaeT2GcL0Btw8-iMp-GXh3y8NWXPZbTp9qgq2EA4zM3rp5t7XVWNasDAcXqgRVmaH8bc2-2AGtgvZRzI7U3F1L-elT2HyFHQbW99puQO8J9VqXbdTjq4b_IsyVc3CtLMDDGU0MU6_02mxaVA5Ll_GadtNts0kay5UcrhVhiEjXqkpSIo_FTag5RfASXKC9eOuy4q8nsDuwuYhTzS8iZbQWX3TvpI5cujvzqPG4OwMcQjoAvtM751VD6qDfLNqaIAZUvZbOyWyHDJ9YGWsue9VugNItWBFdS9WDBsQIMWYJ4Ldw21JVpyEIIsN7o6yYY7Eu9LuFiNWzwENJzB0o9iMpNLhvKofmL2XQxX_imI3dX7G1Oj6bcAnpcPx69toE8KvgHRo0wZcLqGasOlWFC1_N5ym3Yn923ZDFE4H9jN7vaTwqCa1yLsXcn5T6cRNvKYm8S8X9q8Hce1GwbgenWTlujadgLavBiAh7HIMtzSQmnxa_bl4T8xzaEoP-PxF-gRhJlatQJml6NT9abgKnB3DXTKGAT0vBg62JGT7_V0IUlknpIrh_04_t08eoMHsN_Cv9OMiq4RYtVvcNF3Bo7YhKiIvo1EcSRsN3GCH1_XN582gNTVPnHrAyJIKxpadhHgm2-yksn5XK7o44gSnUzeHy25PsZL1mbJVjDvnhrwXyzufhimJ5CiOfjMGtdWlLuEomYBNJAmKF6t0lMMAmWLbAHu7uRgLwoT3HRK8zdQjf1KB2ytre3oj5_yE71ihC3m1kmhYNChbGeqY9xD2VKXM2qWL5EjXwtTtMSsKI_5CjUG0vc7Xm1avj9-CBgyi_SlUvfMovIOLU2uA7Iia9g07vyCv2CucRM&isca=1)

Reply:
@zli82016 pushed the fix.